CONTRIBUTING.md

```diff
@@ -1,38 +1,68 @@
 # Contribution Guidelines
 
 ## Raising an Issue
+
 If you raise an issue against this repository, please include as much information as possible to reproduce any bugs,
 or specific locations in the case of content errors.
 
 ## Contributing code
+
 To contribute code, please fork the repository and raise a pull request.
 
 Ideally pull requests should be fairly granular and aim to solve one problem each. It would also be helpful if they
 linked to an issue. If the maintainers cannot understand why a pull request was raised, it will be rejected,
 so please explain why the changes need to be made (unless it is self-evident).
 
 ### Merge responsibility
-* It is the responsibility of the reviewer to merge branches they have approved.
-* It is the responsibility of the author of the merge to ensure their merge is in a mergeable state.
-* It is the responsibility of the maintainers to ensure the merge process is unambiguous and automated where possible.
+
+- It is the responsibility of the reviewer to merge branches they have approved.
+- It is the responsibility of the author of the merge to ensure their merge is in a mergeable state.
+- It is the responsibility of the maintainers to ensure the merge process is unambiguous and automated where possible.
 
 ### Branch naming
+
 Branch names should be of the format:
 
 `aea-nnn-short-issue-description`
 
 Multiple branches are permitted for the same ticket.
 
 ### Commit messages
+
 Commit messages should be formatted as follows:
-```
+
+```text
 AEA-NNN Summary of changes
 
 Longer description of changes if explaining rationale is necessary,
 limited to 80 columns and spanning as many lines as you need.
 ```
```

Comment:
Is it worth including commit message standards if we always squash merge PRs?
We don't have an assurance system in place if it's a standard we want to start using